libs/media-player/src/audio-annotator.cy.tsx

```diff
@@ -0,0 +1,352 @@
+            it(`should be disabled`, () => {
+                cy.getByDataAttribute('in-point-marker-button').should('be.disabled');
+
+                cy.getByDataAttribute('out-point-marker-button').should('be.disabled');
+            });
+        });
+    });
+
+    describe(`when there is a valid value for audioUrl`, () => {
+        beforeEach(() => {
+            cy.mount(<AudioAnnotatorWidget audioUrl={validAudioUrl} />);
+        });
+
+        it('should display audio controls', () => {
+            cy.get('audio').should('be.visible');
+
+            cy.get('audio').should('have.prop', 'controls', true);
+        });
+
+        describe(`when the media first loads`, () => {
+            before(() => {
+                cy.wait(1000);
+            });
+
+            describe(`the mark in-point button`, () => {
+                it(`should be enabled`, () => {
+                    cy.getByDataAttribute('in-point-marker-button').should('be.enabled');
+                });
+            });
+
+            describe(`the mark out-point button`, () => {
+                it(`should be disabled`, () => {
+                    cy.getByDataAttribute('out-point-marker-button').should('be.disabled');
+                });
+            });
+        });
+
+        describe(`when the media has been played`, () => {
+            beforeEach(() => {
+                cy.get('audio').then(([audioElement]) => {
+                    audioElement.play();
+                });
+
+                cy.wait(2000);
+            });
+
+            describe(`when the mark in-point button is clicked`, () => {
+                it(`should mark an in-point`, () => {
+                    cy.getByDataAttribute('in-point-marker-button').click();
+
+                    cy.getByDataAttribute('in-point-selection-time-code').should('exist');
+
+                    cy.getByDataAttribute('inpoint-selected-bar').should('be.visible');
+                });
+            });
+
+            describe(`when the keyboard shortcut for mark in-point is pressed`, () => {
+                it(`should mark an in-point`, () => {
+                    cy.get('body').type('i');
+
+                    cy.getByDataAttribute('in-point-selection-time-code').should('exist');
+
+                    cy.getByDataAttribute('inpoint-selected-bar').should('be.visible');
+                });
+            });
+
+            describe(`when the mark in-point button is clicked, the mark out-point button`, () => {
+                it('should be enabled', () => {
+                    cy.getByDataAttribute('in-point-marker-button').click();
+
+                    cy.wait(300);
+
+                    cy.getByDataAttribute('out-point-marker-button').should('be.enabled');
+                });
+            });
+
+            describe(`when the keyboard shortcut for mark in-point is pressed, the mark out-point button`, () => {
+                it('should be enabled', () => {
+                    cy.get('body').type('i');
```

Comment:
I assume we can use the default set of keyboard shortcuts in this test.

Reply:
Yea. That's good for now. We can update this when we support configurable key bindings.